DSL/Ruuter/POST/services/connection-requests.yml

```diff
@@ -11,6 +11,10 @@ get_connection_requests:
   call: http.post
   args:
     url: "[#SERVICE_TRAINING_RESQL]/get-requested-service-triggers"
+    body:
+      page: ${incoming.body.page}
+      page_size: ${incoming.body.page_size}
+      sorting: ${incoming.body.sorting}
```

Comment:
Please maintain the `declaration` step after these changes:

- change `method: get` to `method: post`
- add the new fields to the `allowlist` as in:
```
declaration:
  ...
  allowlist:
    body:
      - field: page_size
        type: number
        description: "Body field 'page_size'"
      - field: sorting
        type: string
        description: "Body field 'sorting'"
      ...etc
```